ovp.LucyBot.yml

```diff
@@ -764,6 +764,10 @@ androidNavigation:
     markdownURL: /assets/documentation/player-portal/android/ads-android.html
     path: "/ads-android"    
     
+  - title: "Playlist Configuration"
+    markdownURL: /assets/documentation/player-portal/android/playlist-android.html
+    path: "/playlist-android"
```

Comment:
Did you add the html asset?

Reply:
yesterday Jess merged it
Can I merge this one?